sentry/templates/NOTES.txt

```diff
@@ -1,3 +1,3 @@
 * When running upgrades, make sure to give back the `system.secretKey` value.
 
-kubectl -n {{ .Release.Namespace }} get configmap {{ template "sentry.fullname" . }}-sentry -o go-template='{{ index .data "config.yml" }}' | awk '$1=="system.secret-key:" {print $2}' | tr -d '"'
```

Comment:
go-template should be the correct ones see the 
https://github.com/sentry-kubernetes/charts/pull/525

Also I tested with my mac only the go -template version works